### DISCUSSION TITLE
Docker images and disk usage

### DISCUSSION THREAD

**Discussion by hstek** [teslamate-org/teslamate#1874] [Q&A] [answered]
I have one car using TM for about 8 months and my /var/lib/docker is nearing 6GB in size.   Docker isn't used for anything else.

I noticed that /var/lib/docker/overlay2 keeps growing in size (normal), buy especially after pulling newer images.

Running a `docker image ls` will show several images of the same  (postgres, mosquitto,grafana) created over time.
Anyone cleaning the mess or automated that ?

**Comment by andrewjw** [comment, marked as answer]
Running `docker system prune` will clean up unused images and containers.
